src/components/modules/blockManager.ts

```diff
@@ -255,13 +257,15 @@ export default class BlockManager extends Module {
    * @returns {Block}
    */
   public insert({
+    id = _.generateBlockId(),
```

Comment:
I think, `id` should be generated in the single place — in the `composeBlock`

Reply:
Yes I completely agree, did this change (and the other similar one) and it even fixed the id being re generated when moving the block up/down